src/game/server/gamemodes/mod.cpp

```diff
@@ -826,6 +826,12 @@ void CGameControllerMOD::DoWincheck()
 		str_format(aBuf, sizeof(aBuf), "round_end winner='zombies' survivors='0' duration='%d' round='%d of %d'", Seconds, m_RoundCount+1, g_Config.m_SvRoundsPerMap);
 		GameServer()->Console()->Print(IConsole::OUTPUT_LEVEL_DEBUG, "game", aBuf);
 
+		if (g_Config.m_InfEnableWitchPortals)
+		{
+			str_format(aBuf, sizeof(aBuf), "round_end_witch_with_portals winner='zombies' survivors='0' duration='%d' round='%d of %d'", Seconds, m_RoundCount+1, g_Config.m_SvRoundsPerMap);
```

Comment:
please switch the position of this message and of the message about, otherwise it breaks stats

Reply:
Hm, yeah, makes sense. 'll do.

Reply:
it seems that we should remove this additional line at all, because it breaks the stats [0]. But it also seems that we can just append something to the `round_end` message. Like `round_end winner='zombies' survivors='0' duration='%d' round='%d of %d' type='new_witch'`

[0] https://github.com/ResamVi/infclass-stats/blob/master/operations/parser.go#L126

Reply:
OK